README.md

```diff
@@ -1,152 +1,85 @@
-#Twitter Cleo App
+# Twitter Cleo App
 
-## Readme format
+### Readme format
 
 The format used to build this readme is  [GitHub Flavored Markdown Spec format](https://github.github.com/gfm/)
 
+### Fork strategy: [Feature flow](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow)
+ - Main branch: main
+ - Feature branch: feature/XXX-XXX-XXX-name-ticket
 
-
-* Roles:
-- * Podruct Owner
-- * Technical Lead
-- * Facilitator
-
```

Comment:
I think it is. not fork strategy is branch strategy xd

Reply:
It was deleted.